### ISSUE TITLE
General review of Limpet

### ISSUE THREAD

**Issue by IanMayo** [debrief/limpet#260]
@bcpeters3 - has joined the project, and he has great UI/data dashboard experience.

This will be to go through the "Limpet Requirements" and "Getting started with Limpet" documents from the Limpet home page:   http://limpet.info

Feedback is welcomed in these areas:
- General approach
- User interface
- Language & terms used

**Comment by bcpeters3** [comment]
Hi Ian - Can I get access to this Google doc so I can make notes and suggestions?
https://docs.google.com/document/d/1Vgz7By9KxAhr1s_LmRSwKqtucB_RFC6Ovrn5yk4wznc/pub

For now, I will track everything in my own document and port the changes over later.
Cheers,
Brandon

**Comment by bcpeters3** [comment]
Hi Ian,

I've reviewed the Getting Started with Limpet word document. Overall, I found it to be an excellent introduction to Limpet's features. 

My feedback was primarily around the questions that the document led me to ask (e.g. what other capabilities are there here? How would this work in a more general sense?) and some clarification around wording. I imagine a lot of these questions would be answered as the documentation continues to grow. I also found some of the screenshots did not match exactly, so I added the ones I generated in case they should be updated from an older version (mostly just the color/style).

The biggest question I still had after working through the document was WHY to do something (the document addressed HOW very well). This may mostly be due to me being new to Limpet (and not knowing much about sailing), but I still felt I lacked some context as to why something was being done to the data and how I would take this tool and apply it to my own data sets. I don't think this needs to be explained in the current document, but perhaps somewhere else. Let me know if this makes sense.

I started looking through the Intro to Limpet slides as well, but I wanted to clarify their purpose/audience before digging into those. Once you let me know how these will be used, I will add my feedback as well.

Cheers,
Brandon

**Comment by IanMayo** [comment]
Morning Brandon,
All good points.

Yes, the sailing analysis is a bit contrived, hopefully our automotive
datasets will let us demonstrate something more accessible.

The limpet slides were produced as a vehicle to get feedback on the
original aspirations & requirements. They may form part of an
introduction/background to limpet in the future (especially the fancy
"features" flowchart).

**Comment by bcpeters3** [comment]
Hi Ian,
I've added comments to the slides as well. Please let me know if you have any questions about the feedback.

**Comment by IanMayo** [comment]
Great, thanks. I can see your comments on the getting started doc - but I can't see any on the slides.  Could you give me the slide number of a slide that you've commented on?  I'll double-check that the comment is in there.

**Comment by bcpeters3** [comment]
Sorry not sure what happened there. Comments should be there now.

**Comment by IanMayo** [comment]
Yes, have replied to all comments. If you have any outstanding questions/observations, please add them to each comment "thread", or "Resolve" the comment if relevant.

thanks,
Ian

**Comment by IanMayo** [comment]
Hi there @bcpeters3, I haven't forgotten you :-)

The development is moving ahead, and we're now plotting composite datasets:
![firststackedcharts](https://cloud.githubusercontent.com/assets/1108513/15327073/21897cb4-1c48-11e6-9444-f18a09f9c4ec.png)

We've also got some rudimentary support for the editor view, though it still needs to catch up with the viewer:
![initial_editor](https://cloud.githubusercontent.com/assets/1108513/15327131/4cfecc32-1c48-11e6-9c81-89845a4dafb1.png)

I'm currently preparing for a demonstration tomorrow morning.  But, in the next few days I hope to describe the _use cases_ for this particular set of graphs, and to gather some feedback from you.

Cheers,
Ian

**Comment by bcpeters3** [comment]
No worries Ian - just let me know when you'd like me to review.

Cheers,
Brandon

**Comment by IanMayo** [comment]
I've been thinking over the last couple of days about the "human factors", or usability side of the charts.

In the specific instance shown above, analysts are trying to gain an understanding for what actually happened between two ships/submarines/whatever.

[This article](https://www.oreilly.com/ideas/using-design-patterns-to-find-greater-meaning-in-your-data?imm_mid=0e3d2e&cmp=em-data-na-na-newsltr_20160518) presents "design patterns" for understanding data.  These design patterns may represent a "language" we can use to consciously allow analysts to understand their data.

I think we have a couple more days of doing some obvious fixes to the data, then I'd like to give you a "use case" about the graphs being used, then give you some time to think about tools/processes we can use to extract meaning.

cheers,
Ian

**Comment by IanMayo** [comment]
Hi there @bcpeters3 - sadly I've run out of time to collate a copy of the software for you to start playing with, and providing feedback.  I still need to create a way for analysts to create the charts from data (#334)
